provider/cmd/pulumi-resource-materialize/schema.json

```diff
@@ -8367,6 +8381,10 @@
                     "$ref": "#/types/materialize:index/SinkKafkaFrom:SinkKafkaFrom",
                     "description": "The name of the source, table or materialized view you want to send to the sink.\n"
                 },
+                "headers": {
+                    "type": "string",
+                    "description": "The name of a column containing additional headers to add to each message emitted by the sink. The column must be of type map[text =\u003e text] or map[text =\u003e bytea].\n"
```

Comment:
Will the `\u003e` be  rendered as a `>` in whatever renders this?

Reply:
Yep, the `\u003e` part should be rendered to `>` and look as follows: `map[text => text]`.

The schema.json is actually auto-generated using the `make tfgen` command, so it was the `pulumi/pulumi-terraform-bridge` which did that translation.